library/src/main/java/com/chuckerteam/chucker/internal/ui/error/ErrorViewModel.kt

```diff
@@ -0,0 +1,25 @@
+package com.chuckerteam.chucker.internal.ui.error
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import com.chuckerteam.chucker.internal.data.entity.RecordedThrowable
+import com.chuckerteam.chucker.internal.data.repository.RepositoryProvider
+
+internal class ErrorViewModel(
```

Comment:
nit: I would rename this to `ThrowableViewModel`, let's try to be consistent with the naming. I noticed that we have a mixture of `Error` and `Throwable` in the codebase. I would stick to only one. Given that `Throwable` is what we use for the database, we should probably use it and apply it also to `ErrorActivity`, `ErrorViewModel`, etc. (so they should be `ThrowableActivity` and `ThrowableViewModel`).

I'm anyway not strongly opinionated on this front, so we can tackle this in the future as well.

Reply:
Absolutely agree here. Also was regularly confused by mixing `error` and `throwable`. Will update the PR.